src/libslic3r/GCode.cpp

```diff
@@ -4545,8 +4545,13 @@ std::string GCode::_extrude(const ExtrusionPath &path, std::string description,
              (m_config.get_abs_value("overhang_4_4_speed") < 0.5) ?
                  FloatOrPercent{100, true} :
                  FloatOrPercent{m_config.get_abs_value("overhang_4_4_speed") * 100 / out_wall_ref_speed, true},
-             FloatOrPercent{m_config.get_abs_value("bridge_speed") * 100 / out_wall_ref_speed, true},
-             FloatOrPercent{m_config.get_abs_value("bridge_speed") * 100 / out_wall_ref_speed, true}});
+             (m_config.get_abs_value("overhang_4_4_speed") < 0.5) ?
```

Comment:
Hi @igiannakas 
This is behavor is intentional.
If a perimeter is not more than 90 percents supported, it is considered as bridge.
If you look at the overhang speed setting UI, it mentions this.

Reply:
As of the 13 percents here in the code, this is the magic number I come up with afte comparing/tuning many many models.

Reply:
Understood. Should we explore the opportunity to scan for curled edges and use that as a mechanism to further refine the slow down for overhangs algorithm? What do you think?

Reply:
Sure! 😀